python/kserve/kserve/handlers/list.py

```diff
@@ -0,0 +1,10 @@
+from kserve.handlers.base import BaseHandler
+from kserve.model_repository import ModelRepository
+
+
+class ListHandler(BaseHandler):
```

Comment:
I think `ListHandler` and `LoadHandler` can be merged into `model_management.py`